src/hooks/api/useItemNotesEsc.js

```diff
@@ -0,0 +1,16 @@
+import {useNotesEsc} from "./useNoteEsc";
+import {OPTIONS, PARAMETERS_KEYS} from "../../constants";
+
+export const useItemNotesEsc = (tenants, type, options = {}) => {
+  return useNotesEsc(
+      'item-note-types-esc',
+      tenants,
+      type,
+      'ui-bulk-edit.category.itemNotes',
+      options,
+      'item-note-types',
+      'itemNoteTypes',
```

Comment:
Looks like this one should be in constants

Reply:
done!